docs/examples.rst

```diff
@@ -394,3 +427,6 @@ issues.
 .. _GNU Parallel: https://www.gnu.org/software/parallel/
 .. _Pytest: https://docs.pytest.org/en/latest/
 .. _UUID: https://en.wikipedia.org/wiki/Universally_unique_identifier
+
+[^1]: [Implementation in molecule-docker](https://github.com/ansible-community/molecule-docker/blob/f4efce3c4fda226c8ca5f10976927fff7daa8e69/src/molecule_docker/playbooks/create.yml#L35)
+[^2]: e.g. [Debian uses `sudo` instead of `wheel` group.](https://wiki.debian.org/sudo/)
```

Comment:
And the footnote body should then look like this:

```
.. [#dockerimplementation]: `Implementation in molecule-docker <https://github.com/ansible-community/molecule-docker/blob/f4efce3c4fda226c8ca5f10976927fff7daa8e69/src/molecule_docker/playbooks/create.yml#L35>`_)
.. [#sudogroup]: e.g. `Debian uses `sudo` instead of `wheel` group. <https://wiki.debian.org/sudo/>`_
```

Note, that [inline (external) web links](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-link) also look different in SphinxDoc/rST. I also updated the links.